doc/pages/tutorials/010-astarte_in_5_minutes.md

```diff
@@ -6,10 +6,45 @@ This tutorial will guide you through bringing up your Astarte instance, creating
 
 First of all, please keep in mind that **this setup is not meant to be used in production**: by default, no persistence is involved, the installation does not have any recovery mechanism, and you will have to restart services manually in case something goes awry. This guide is great if you want to take Astarte for a spin, or if you want to use an isolated instance for development.
 
-You will need a machine with at least 4GB of RAM, with [Docker](https://www.docker.com/) and [astartectl](https://github.com/astarte-platform/astartectl) installed. If you don't have `astartectl` installed on your machine yet, you should install it by following the instructions in [astartectl's README](https://github.com/astarte-platform/astartectl#installation)
+You will need a machine with at least 4GB of RAM, a recent 64-bit operating system with [Docker](https://www.docker.com/) and [astartectl](https://github.com/astarte-platform/astartectl) installed. If you don't have `astartectl` installed on your machine yet, you should install it by following the instructions in [astartectl's README](https://github.com/astarte-platform/astartectl#installation)
```

Comment:
While we're at it, mention `docker-compose` explicitly (it's a separate package from Docker in, e.g., Ubuntu)